tests/sql/moz-fx-data-shared-prod/search_derived/mobile_search_clients_daily_v1/test_overactive_filter/org_mozilla_ios_fennec.metrics.schema.json

```diff
@@ -60,6 +60,40 @@
             "mode": "REPEATED",
             "name": "search_in_content",
             "type": "RECORD"
+          },
```

Comment:
For completeness I might add some actual test cases for these new columns which test this behaviour (overactive filtering).

If I understand the code correctly they should be filtered out using the same mechanisms.

Reply:
Added!